src/Docs/Search/Types.purs

```diff
@@ -0,0 +1,5 @@
+module Docs.Search.Types where
+
+type ModuleName = String
```

Comment:
Maybe we could consider using newtypes here..

Reply:
I'll do it in a separate PR. The best set of types here would be:

```
newtype ModuleName = ModuleName String
newtype PackageName = PackageName String
data PackageInfo = Package PackageName | Builtin | LocalPackage | UnknownPackage
```